angular/src/app/project-form/project-form.component.ts

```diff
@@ -48,8 +43,9 @@ export class ProjectFormComponent implements OnInit {
 
   onDelete() {
     this.deleted = true;
-    this.projectFormService.delete({'id': this.project._id}).subscribe((response) => {
+    this.projectFormService.delete({'id': this.project.id}).subscribe((response) => {
       console.log(response);
```

Comment:
Here

Reply:
What needs to be changed here?

Reply:
remove the console log statement here as well